README.md

```diff
@@ -92,3 +93,44 @@ More useful commands to execute from the `data-science-101` root directory
 - `vagrant destroy -f && vagrant up`
   - if things get SUPER messed up and you need to start all over
 
+
+## Alternative setup using docker 
+
+### Step 1
+
+Install:
+- [git](https://git-scm.com/)
+  - If you're on Windows or OS X and don't know what git is, [GitHub Desktop](https://desktop.github.com/) is probably easiest.
+  - All others: you probably know what to do.
+- [docker](https://docs.docker.com/engine/installation/)
+
+### Step 2
+
+If you installed git using `GitHub Desktop`, follow [these instructions](https://help.github.com/desktop/guides/contributing/cloning-a-repository-from-github-to-github-desktop/)
+
+If you are cloning from the command line:
+```shell
+1. $ git clone https://github.com/hershaw/data-science-101.git
+```
+
+### Step 3 
+
+1. Build a docker image (first time takes longer):
+```shell
+    $ docker build -t data-science -f Dockerfile .
+```
+2. Run a container:
+```shell
+    $ docker run -it --rm -p 8888:8888 --volume ~/workspace/data-science-101:/data-science-101 \
+    --workdir /data-science-101 data-sciencecience-101 -e -e PYTHONPATH=.:/data-science-101/course data-science
+```
+
+### Step 4
+
+Open the link provided in the interactive shell something like http://localhost:8888/?token=d1ab96467698d1addffac95d634d777fd059ccd4a7b2094e.
```

Comment:
Can you adjust the jupyter config file the same as done in the [Vagrantfile](https://github.com/hershaw/data-science-101/blob/master/Vagrantfile#L24) so that you don't need the token when opening the notebook.